aries-site/src/examples/foundation/branding/HpeElementExample.js

```diff
@@ -7,7 +7,7 @@ export const HpeElementExample = () => {
 
   return (
     <Box direction="row" align="center" gap="medium">
-      <Hpe color="brand" size="66px" />
+      <Hpe color="brand" size="large" />
```

Comment:
Think we don't want any size here, but want to use the default size
```suggestion
      <Hpe color="brand" />
```